src/pages/QuestionListsPage/QuestionLists.tsx

```diff
@@ -67,8 +66,14 @@ export default function QuestionLists(): JSX.Element {
     setModalVisibility(false);
   };
 
-  const tags = (interviewQuestions: QuestionModel[]): JSX.Element[] => {
-    return getDistinctCategories(interviewQuestions).map((category) => (
+  const deleteList = (id: number): void => {
+    client.deleteQuestionSet(id).then(() => {
+      setLists(lists?.filter((item) => item.questionSet?.id !== id));
+    });
+  };
+
+  const tags = (categories: string[]): JSX.Element[] => {
```

Comment:
I am wondering if this is a real way how it should be done. I would expect elements  <Tag> to be generated inside the card from categories(strings) received as props. If you say this is the way I trust you 😊

Reply:
Yes, that would be better, that's on me since I implemented it this way 😅